.buildkite/macOS.yml

```diff
@@ -8,3 +8,4 @@ steps:
       EARTHLY_BUILD_ARGS: "DOCKERHUB_USER_SECRET=+secrets/earthly-technologies/dockerhub/user,DOCKERHUB_TOKEN_SECRET=+secrets/earthly-technologies/dockerhub/token"
     agents:
       os: macOS
+      arch: amd64
```

Comment:
Had to since the mac m1 build isnt 100 percent right now anyways... we way want to consider splitting into smoketests per os/arch and let the long test fall where it may